.github/workflows/update-pinned-reqs.yml

```diff
@@ -32,10 +32,33 @@ jobs:
 
     - name: Create Pull Request
       uses: peter-evans/create-pull-request@v5
+      env:
+        GITHUB_TOKEN: ${{ secrets.REPO_SCOPED_TOKEN }}
```

Comment:
This line is so that the GitHub Actions will actually run by an auto-generated PR.

Reply:
👍